native/packages/react-native-bpk-component-button/src/BpkButton.js

```diff
@@ -150,6 +204,16 @@
    selected: PropTypes.bool,
    accessibilityLabel: PropTypes.string,
    style: View.propTypes.style,
+   theme: PropTypes.shape({
+     gradientStartColor: PropTypes.string.isRequired,
```

Comment:
I think we might want a custom prop validation here that looks at the type of the button before determining what is required

Reply:
@k0nserv @shaundon Yes, lets remove the ability to theme disabled styles. Sorry my bad as its on the card.

Reply:
I'm also tempted to ditch the selected button state from native all together as its not being used and its a strange one which we should probably not be promoting. Maybe raise a separate card for this?

Reply:
Happy to remove both selected and disabled states - means less logic and fewer lines of code on our part 😄 

I think we should make a card to remove selected from the existing button, and in the meantime I'll remove selected and disabled from theming.